Replace the expression for calculating a random port number, which
causes an arithmetic error on ash, with a simpler expression which
should also get the job done.

diff -upr gnutls-3.5.13.orig/tests/scripts/common.sh gnutls-3.5.13/tests/scripts/common.sh
--- gnutls-3.5.13.orig/tests/scripts/common.sh	2017-08-22 20:43:46.767748915 +0200
+++ gnutls-3.5.13/tests/scripts/common.sh	2017-08-22 20:47:45.474435430 +0200
@@ -61,7 +61,7 @@ check_if_port_listening() {
 # Find a port number not currently in use.
 GETPORT='rc=0; myrandom=$(date +%N | sed s/^0*//)
     while test $rc = 0;do
-	PORT="$(((($$<<15)|$myrandom) % 63001 + 2000))"
+	PORT="$(($((myrandom + $$)) % 63001 + 2000))"
 	check_if_port_in_use $PORT;rc=$?
     done
 '
